### DISCUSSION TITLE
Would appreciate any advice/help.

### DISCUSSION THREAD

**Discussion by Deja5** [Aelto/W3_RandomEncounters_Tweaks#193] [Q&A]
I've been trying to figure it out on my own the past couple weeks to no avail. I can't seem to get RER to work with w3ee and w3ee redux. I've tried both mods separately first to make sure I wasn't messing up installation with one or the other, but the game starts up fine when I have them installed separately. I followed the in-depth guides merging instructions correctly. The pictures below show redux is not in the mod manager and that RER is not active. That is intentional as during this time I was trying to drop the w3ee redux script folder into the w3ee folder and overwriting the script folder in there and the RER mod I just manually installed by clicking and dragging as well.

edit: forgot to mention I'm on version 4.0 of the game

Mods:
![image](https://user-images.githubusercontent.com/130565195/231452654-43adccb0-ccc3-464b-9409-b045fdc532fd.png)
Merges:
![image](https://user-images.githubusercontent.com/130565195/231452732-cd2b0017-005f-4fe6-ad2e-0b6130d5a9f2.png)

this is the current error that I keep ending up with 
![image](https://user-images.githubusercontent.com/130565195/231453764-433f88cf-011f-48aa-8e1a-ad87922561fa.png)

**Comment by Aelto** [comment]
Hello, i would recommend downloading the mod from Github or using the latest install script. I notice you have bootstrap in your load order but none of your mod (RER included since NextGen) need it.

Something i didn't mention but assumed when writing the guide: I expect you to have W3EE and W3EEREdux installed, with W3EE's script overwritten by Redux ones. That's why the merge instructions only show W3EE. If you prefer them separated then you should merge RER scripts (or the sharedutils) against W3EE Redux' scripts, not W3EE's where, when available.

**Reply by Deja5** [reply]
Hello thank you for replying! I've just unchecked the bootstrap mods you listed and am sure I have downloaded the most recent version of RER from your github. According to the instructions the only thing I needed from the redux zip is the scripts folder right? I just needed to grab that and overwrite the w3ee's script folder ? I believe I have already done that as well. Also I should mention that when merging any conflicts that show up in the script merger I usually choose the sharedutils over w3ee. Like in the picture below I would choose B if a conflict between the 2 emerge. Is that what I'm suppose to do ?
![image](https://user-images.githubusercontent.com/130565195/231635077-af282463-d061-435d-8032-e3e6d6193ba2.png)
